src/gui/userstatusselectormodel.cpp

```diff
@@ -177,7 +178,7 @@ void UserStatusSelectorModel::clearError()
 
 void UserStatusSelectorModel::setOnlineStatus(UserStatus::OnlineStatus status)
 {
-    if (status == _userStatus.state()) {
+    if (!_userStatusConnector || status == _userStatus.state()) {
```

Comment:
I tried adding a Q_ASSERT to this method and already had to remove it because it bombed the entirely application when an early return was already preventing the crash. I agree and would remove the Q_ASSERTs